go.mod

```diff
@@ -11,13 +11,16 @@ require (
 	github.com/alecthomas/participle v0.2.1
 	github.com/aliyun/aliyun-oss-go-sdk v0.0.0-20190307165228-86c17b95fcd5
 	github.com/aws/aws-sdk-go v1.20.21
+	github.com/baiyubin/aliyun-sts-go-sdk v0.0.0-20180326062324-cfa1a18b161f // indirect
```

Comment:
I just removed most of these in #8197 and they don't seem to be needed.

Could you check what is adding them? I use Go 1.13 if that makes a difference.